### PR TITLE
Added Flash Messages and fixed font-awesome link

### PR DESCRIPTION
**Pull request by nykka**
This PR
- Adds four flash messages (error, warning, success, information).
screen shot 2019-03-08 at 12 44 29

----------

- I've used the error message from the Ombulabs page as the base.
screen shot 2019-03-08 at 12 25 00
- I have chosen a different icon for each one.
- **I also have changed the Font Awesome stylesheet link because it was not working anymore, instead I've linked a new version so I had to change de `fa` names.**

### CODE REVIEW

File: compiled/main.css.map [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (lubc) [inline compiled/main.css.map]: @nykka Do we need this file?

Comment (nykka) [reply compiled/main.css.map]: This file is automatically created when run `rake sass`

**Comment by nykka** [comment]
Done.

**Comment by bronzdoc** [comment]
@nykka is this still relevant? could you share images of the new flash messages?

**Comment by nykka** [comment]
@bronzdoc 
Screen Shot 2019-12-04 at 11 12 49

**Pull request merged**